src/styles/blocks/catalog.scss

```diff
@@ -18,6 +18,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition-duration: $hover-timing;
```

Comment:
specify properties you need to apply transition to

Reply:
use transition shorthand